README.md

```diff
@@ -10,7 +10,7 @@ Requires `aws` provider >= 1.36.0
 
 ```hcl
 module "session-manager-settings" {
-  source  = "gazoakley/session-manager-settings/aws"
+  source  = "darkscreenio/session-manager-settings/aws"
```

Comment:
Can you revert the `source` back please?

Reply:
I used this repo for create a new module on terraform, because I was getting close my dead line for solve one question on my job, I'll close this PR and I'll send a new with other branch ok?

Reply:
Sure - thanks :+1: